src/Error/BaseErrorHandler.php

```diff
@@ -149,7 +149,14 @@ public function handleError($code, $description, $file = null, $line = null, $co
                 'path' => Debugger::trimPath($file)
             ];
         }
-        $this->_displayError($data, $debug);
+
+        $shouldDisplay = (
+            $error !== 'Deprecated' ||
+            ($error === 'Deprecated' && $this->_options['displayDeprecations'])
```

Comment:
why `Deprecated` ?
 and  only `E_USER_DEPRECATED`

Reply:
This value is the result of mapping PHP constants to string values.

Reply:
I know but this will  cover both deprecated warnings
E_USER_DEPRECATED + E_DEPRECATED

Reply:
yes.